roles/installer/defaults/main.yml

```
@@ -278,10 +278,10 @@ init_container_extra_commands: ''
 init_container_extra_volume_mounts: ''
 
 ee_images:
-  - name: AWX EE (latest)
-    image: quay.io/ansible/awx-ee:latest
+  - name: "AWX EE ({{ _image_version }})"
+    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
```

maybe change to
```suggestion
  - name: "AWX EE (latest)"
    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
```
to preserve old behavior?

---

with this suggestion it would not create new EE entry, but update the "latest" one. I guess this is exactly the behaviour that I wanted to prevent - that suddenly EE image changes without notice. By using verisoned ones one could set the default on ogranisation level and thus avoid unexpected changes when doing updrade. And after the upgrade is done, due to new EE slot with current version it would make it possible to slowly test all job templates against the new EE and eventually update Organisation to switch everything to the highest version.

Applying the suggestion the moment after upgrade everything would run with different EE without even simple possibility to downgrade. One difference from using literaly "latest" tag is - this switch would happen only when upgrading, reviously it happened after each new release showed up.